lib/tdf3/src/client/builders.ts

```diff
@@ -203,6 +203,10 @@ class EncryptParamsBuilder {
    * @return {EncryptParamsBuilder} - this object.
    */
   withStreamSource(readStream: ReadableStream<Uint8Array>): EncryptParamsBuilder {
+    if (!inBrowser() && !readStream?.getReader) {
+      readStream = toWebReadableStream(readStream);
+    }
+
```

Comment:
The type here is ReadableStream, so just assert that it is if you are worried about using this method outside of a TS context
```suggestion
    assert readStream.getReader;
```